index.html

```diff
@@ -1,48 +1,79 @@
 <!DOCTYPE html>
 <html lang="en">
-  <head>
-    <title>T8 Character Moves Rant</title>
-    <!-- <meta name="description"> T8 Moves Rant/Guide </meta> -->
-    <meta charset="UTF-8" />
-    <meta name="viewport" content="width=device-width" />
-    <!--
+
+<head>
+  <title>T8 Character Moves Rant</title>
+  <!-- <meta name="description"> T8 Moves Rant/Guide </meta> -->
+  <meta charset="UTF-8" />
+  <meta name="viewport" content="width=device-width" />
+  <!--
       Need a visual blank slate?
       Remove all code in `styles.css`!
     -->
-    <link rel="stylesheet" href="styles.css" />
-    <script type="module" src="script.js"></script>
-    <link rel="icon"
-    href="/images/favicon.ico">
-  </head>
-  <body>
-        <nav>
-          <a href="/" aria-current="page">Home</a>
-          <a href="/page2.html">Jin Kazama</a>
-          <a href="/page3.html">Kazuya Mishima</a>
-          <a href="/page4.html">Renia</a>
-          <a href="/page5.html">Heihachi Mishima</a>
-        </nav>
-    <main>
-    <h1>T8 Character Rant</h1>
-      <div>
-         <br>
-        <h2> Feature Characters</h2><b> 
-          <em>
-        <a href="page2.html"> Jin Kazama</a> 
-        <br>
-        <a href="page3.html"> Kazuya Mishima</a>
-        <br>
-        <a href="page4.html"> Renia</a>
-        <br>
-        <a href="page5.html"> Heihachi Mishima</a>
-        </div>
-            <br>
-      <h1>Trailer</h1>
-      <div>
-      <iframe width="720" height="720" src="https://www.youtube.com/embed/_MM4clV2qjE" title="TEKKEN 8 – OFFICIAL LAUNCH TRAILER" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share" referrerpolicy="strict-origin-when-cross-origin" allowfullscreen></iframe>
+  <link rel="stylesheet" href="styles.css" />
+  <script type="module" src="script.js"></script>
+  <link rel="icon" href="/images/favicon.ico">
+  <script src="https://kit.fontawesome.com/be6975ecda.js" crossorigin="anonymous"></script>
+</head>
+
+<body>
+  <nav>
+    <ul>
+      <li><a href="#">Home</a></li>
+      <li>
+        <a href="#">Characters</a>
+        <ul class="dropdown">
+          <li><a href="#">Jin Kazama</a></li>
+          <li><a href="#">Kazuya Mishima</a></li>
+          <li><a href="#">Renia</a></li>
+          <li><a href="#">Heihachi</a></li>

```

Comment:
you can change the `href`s here to go to pages 2-5 if you want these to link to somewhere besides the homepage. For example, `<a href="/page2.html">Jin Kazama</a>`